### ISSUE TITLE
'cellEdited' cell callback is not working

### ISSUE THREAD

**Issue by bornova** [olifolkerd/tabulator#3536]
**Describe the bug**
`cellEdited` callback of the cell component is not called after a cell is edited.

Reference: http://tabulator.info/docs/5.0/callbacks#cell

**Tabulator Info**
- 5.0.10

**Working Example**
https://jsfiddle.net/izmirli/bLyzpr3o/

**To Reproduce**
A step by step guide to recreate the issue in your JS Fiddle or Codepen:
1. Go to https://jsfiddle.net/izmirli/bLyzpr3o/
2. Click on any 'age' cell to edit
4. Notice `cellEditing` and `cellEditCancelled` is called but not `cellEdited`

**Expected behavior**
`cellEdited` calback is called after editing a cell.

**Comment by AndrewHutcheson** [comment]
+1
I also have this problem after upgrading from v4.9

**Comment by alwaysnandan** [comment]
I am also having this error after updating from version 4.x to 5.0 this morning.

**Comment by H3le** [comment]
+1

Also having that issue, i opened a question on SO a coupple of week ago.
I wasn't using the subscription model, my bad, but this event is not working even after using it (Tabulator version is 5.0.10).
https://stackoverflow.com/questions/70564925/tabulator-v5-0-10-is-logging-stuff-in-console-instead-of-firing-expected-callbac/70653197?noredirect=1#comment124932271_70653197

**Comment by H3le** [comment]
Hi,  
let me take a step back.

cellEdited callbacks is actually working for me under the subscription module, i was just not referencing the right table object, stupid me. The following piece of code is working for me
```javascript
        var Table= new Tabulator("#table", {
            data: [{"Name": "Jhonny", "Age" : 25}],
            layout: "fitColumns",
            pagination: "local",
            responsiveLayout: "collapse",
            columns: [
                { title: "Name", field: "Name", headerFilter: true, editor: "input" },
                { title: "Age", field: "Age", headerFilter: true, editor: "input" },
            ]
        })

        Tabella.on("cellEdited", function (cell) {
            alert("HERE")
        });
```

@bornova i noticed that you are not using the subscription model as reported here  
http://tabulator.info/docs/5.0/events#cell

Is your fiddle expected to work as is it written?
With the code above all cells are triggering the event as expected.
Cheers,
Hele.

**Comment by bornova** [comment]
Hi @H3le,
I am aware `cellEdited` callback works as expected when used as a table event.  Problem is it doesn't work when used at cell level as documented here: http://tabulator.info/docs/5.0/callbacks#cell

**Comment by H3le** [comment]
Ok, that was just to be sure!
In docs is reported that some callbacks where **moved** to events like cellEdited, so my mind went "wait moved means no longer available where it was before".

But as you linked it should still be available to be used as before, so ja, thanks for the clarification.

**Comment by ivo-bloechliger** [comment]
Hi there
First note to self: study the issues before "fixing" some code. Anyway. The current documentation at http://tabulator.info/docs/5.0/callbacks#cell lists the following example:

> {title:"Name", field:"name", cellEdited:function(cell){
>     //cell - cell component
>     },
> }

which I found most intuitive. But it did not work. I created a pull request (re?)enabling this event. In the hope that helps.
Warm regards
Ivo

**Comment by KAYFLUHR** [comment]
Hi! I also had a lot of problems with version 5.0.10. I found out that this patter worked for me:

this.tabulator.on("cellClick", (e, cell)=> this.setSelection(e, cell, this.tabulator));

where this.setSelection is a method in the method secion.

1 more example:
this.tabulator.on("tableBuilt", ()=> this.getAllTabulatorFields(currentTab, this.showBusinessView));

**Comment by olifolkerd** [comment]
Hey Everyone,

I have just merged in a PR to master to fix this, it will be included in sundays 5.1 release.

Cheers

Oli :)

**Comment by ValEmpire** [comment]
Same error on react-tabulator
